### PR TITLE
added active status for Economy cards

### PR DESCRIPTION
**Pull request by ksenia-b**

### CODE REVIEW

File: client/src/containers/Economy/components/GDP_PCAP/index.tsx [hunk 1]
```diff
@@ -7,16 +7,17 @@ import { Fetch } from '../Fetch';
...
+export const GDPperCapita = ({ onData, isActive }) => {
   const { t } = useTranslation();
   const title = t('economy.GDP_per_capita');
 
   const onClick = () => {
     onData({
       title,
       unit: '$',
-      indicator: WORLD_BANK_INDICATOR.GDP_PER_CAPITA,
+      indicator:'myindicator',
```
Comment (MaksymRudnyi) [inline client/src/containers/Economy/components/GDP_PCAP/index.tsx:18]: Revert to the correct value.

**Review by MaksymRudnyi — APPROVED** [review]

File: client/src/containers/Economy/index.tsx [hunk 2]
```diff
@@ -26,7 +26,10 @@ export const Economy = () => {
     setTitle(title);
     setUnit(unit);
   };
-
+  
+  const isElementActive = (el:string) => {
```
Comment (MaksymRudnyi) [inline client/src/containers/Economy/index.tsx:30]:
```suggestion
  const isCardActive = (cardName:string) => cardName === indicator
```

**Comment by ksenia-b** [comment]
[Active state for Economy section](https://trello.com/c/h82etEdU/18-active-state-for-economy-section)

File: client/src/containers/Economy/index.tsx (hunk 2 above)
Comment (ksenia-b) [reply client/src/containers/Economy/index.tsx:30]: done

**Pull request merged**